MAUI/ImageEditor/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Overview of .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control, its basic features and functionalities of imageeditor.
+platform: .Net MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Overview of .NET MAUI ImageEditor (SfImageEditor)
+
+The Syncfusion .NET MAUI ImageEditor (SfImageEditor) is a very handy tool, which is used to edit an image by annotating with free hand drawing paths, text, and built-in shapes; it also allows to crop and flip a image. This control has a built-in toolbar that helps to perform editing operations.

```

Comment:
The Syncfusion .NET MAUI ImageEditor (SfImageEditor) is a very handy tool, which is used to edit an image by annotating with freehand drawing paths, text, and built-in shapes; it also allows to crop and flip an image. This control has a built-in toolbar that helps to perform editing operations.

Reply:
content updated